internal/cache/profiles.go

```diff
@@ -174,11 +171,11 @@ func (p *profileCache) ResourceOperations(profileName string, cmd string, method
 	var resOps []models.ResourceOperation
 	var rosMap map[string][]models.ResourceOperation
 	var ok bool
-	if strings.ToLower(method) == getOpsStr {
+	if strings.ToLower(method) == common.GetCmdMethod {
```

Comment:
This is one of changes unrelated to AutoEvents that I mention in my global review comments. At minimum this should be in it's own commit, but really a separate PR is more appropriate here.

Reply:
will do